### DISCUSSION TITLE
Establish a Postgres CDC Connection

### DISCUSSION THREAD

**Discussion by grantcloud** [risingwavelabs/risingwave#6307] [Q&A] [answered]
hey team - I'm trying to standup a CDC connection to PostGres. The docs [here](https://www.risingwave.dev/docs/latest/sql-create-source/) says PostGres CDC is supported - but the [CDC guide](https://www.risingwave.dev/docs/latest/create-source-cdc/) doesn't show an example of establishing that connection to create a source.

Any help would be appreciated. Thanks

**Comment by liurenjie1024** [comment, marked as answer]
Hi, @grantcloud Currently we can't connect to postgre directly and consume its changelog. It requires customer to setup debezium to produce changes into some message such as kafka, and we can consume debezium format message in kafka.

**Reply by grantcloud** [reply]
Good to know - thanks for the helpful response.

**Comment by liurenjie1024** [comment]
We are also heavily working on connecting postgre to consume its cdc directly.

**Comment by neverchanje** [comment]
Hi @grantcloud Here is the document of Postgres CDC: https://www.risingwave.dev/docs/current/ingest-from-postgres-cdc/ We just released this feature in the 0.1.16 version.

**Comment by fuyufjh** [comment]
Direct PostgreSQL CDC Connector will be released in a few days. With this connector, RisingWave can connect to PostgreSQL directly to obtain data from the binlog without starting additional services. See [Using the native PostgreSQL CDC connector](https://www.risingwave.dev/docs/current/ingest-from-postgres-cdc/#using-the-native-postgresql-cdc-connector) for details.
